src/lib/types/projects.ts

```diff
@@ -100,3 +104,32 @@ export interface PublicInfo {
   description: string;
   github: string;
 }
+
+// ------------------------------------------//
+// --------------------V1--------------------//
+// ------------------------------------------//
+export const zProjectInfo = z.object({
+  description: z.string(),
+  github: z.string(),
+  logo: z.string(),
+  name: z.string(),
+  socials: z.array(
+    z.object({
+      name: z.string(),
+      url: z.string(),
+    })
+  ),
+  website: z.string(),
+});
+
+export type ProjectInfoV1 = z.infer<typeof zProjectInfo>;
+
+export const zPublicAccount = z.object({
```

Comment:
also remove export

Reply:
This also used in `services/account.ts`.